website/pages/docs/k8s/helm.mdx

```diff
@@ -380,6 +380,12 @@ and consider if they're appropriate for your deployment.
         "annotation-key": "annotation-value"
       ```
 
+  - `extraEnvironmentVars` ((#v-server-extra-environment-vars (`string: "{}"`) - extraEnvironmentVars
+    extraEnvVars is a list of extra environment variables to set within the stateful set.
```

Comment:
```suggestion
    extraEnvironmentVars is a list of extra environment variables to set within the stateful set.
```
I know this is copy/paste from the values.yaml file, but I say we should fix this typo there also